.github/drafts/16506/16506_number_slider_draft.md

```diff
@@ -0,0 +1,56 @@
+*** Number Slider ***
+
+The Number Slider widget allows users to choose a numerical value from a range of data.
+
+# Properties
+
+Properties allow you to edit the Number Slider widget, connect it with other widgets and customize the user actions.
+
+| **Property** | **Type** | **Description** | **Code Snippet** |
+|--------------|----------|-----------------|------------------|
+| **Min Value** | Formatting | The starting/smallest possible value of the slider. The value may be negative and/or include decimals, however it must always be less than the **Max Value**. _(number)_ | `{{<widget-name>.minValue}}` |
+| **Max Value** | Formatting | The ending/largest possible value of the slider. The value may be negative and/or include decimals, however it must always be greater than the **Min Value**. _(number)_ | `{{<widget-name>.maxValue}}` |
+| **Step Size** | Formatting | The increment by which the user can adjust the slider's value. This increment must be at least 0.1, and can't be less than the **Min Value** or greater than the **Max Value**. _(number)_ | `{{<widget-name>.stepSize}}` |
```

Comment:
Step size can be less than min value. If the min value is 50 and step size is 1, then it's not an error. @jnikhila @spark-c @yaldram